student.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
+    super(age, parent_permission: parent_permission, name: name)
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly refactor your Student `initialize` method to work as expected. I cannot create a student that inherits from the Person class, because you are misplacing the parameters.

  ```
  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
    super(age, name, parent_permission: parent_permission)
    @classroom = classroom
  end
  ```

  ![image](https://github.com/Kwamay/School_Library/assets/36542986/b13fd5f3-638c-4a1b-aaad-8a9f1941b9a6)